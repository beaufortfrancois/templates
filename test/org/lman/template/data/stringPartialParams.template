{{+partials.stringPartialParams_p1 partial:partials.stringPartialParams_p2}}
{{+partials.stringPartialParams_p2}}
