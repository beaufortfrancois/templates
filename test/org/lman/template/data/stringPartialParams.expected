This is the first partial.
Name: Ben Kalman.
This is a template parameter that we'll pass the string to:
  This is the second partial.
  First name: Kalman
  Second name: Ben
This is the second partial.
First name: Ben
Second name: Kalman
